lib/bolt/cli.rb

```diff
@@ -184,6 +197,14 @@ def finalize_setup
 
       warn_inventory_overrides_cli(options)
 
+      # Error if the module command is disabled
+      if options[:subcommand] == 'module' && config.project.modules.nil?
+        raise Bolt::CLIError,
+              "Unable to use command 'bolt module #{options[:action]}'. To use "\
+              "this command, update your project configuration to manage module "\
+              "dependencies."
```

Comment:
This message will need a link to the proper docs once those are written. I left it out for now since I'm not sure what the URL for that page will be.

Reply:
This hits the `validate()` function before getting here, which checks the subcommands on line 237. I think this message needs to be checked and raised there.
```
lucy ~/githubs/bolt ((1ab94b3...)) $ be bolt module help
Expected subcommand 'module' to be one of command, script, task, plan, file, puppetfile, secret, inventory, group, project, apply, guide
```

Reply:
Similarly, it looks like if `bolt module help` or `bolt module --help` is run there's not message that the user needs to enable the module subcommand, which I think would be useful.

Reply:
We can't perform this check as part of `validate()` without changing when/how the command is validated. The validations happens before config is loaded, and we need config to be loaded to check whether `modules` is configured.

Also, the `module` command is still feature flagged in this PR and `help` isn't a valid action for any subcommand, so that may be part of what you're running into? Setting the flag and running a `module` command without `modules` configured will get you to this message and not error during validation.

I have updated the help text for the `module` commands to say that `modules` needs to be configured.